app/sidekiq/lighthouse/evidence_submissions/delete_evidence_submission_records_job.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module Lighthouse
+  module EvidenceSubmissions
+    class DeleteEvidenceSubmissionRecordsJob
+      include Sidekiq::Job
+
+      # No need to retry since the schedule will run this periodically
+      sidekiq_options retry: false
+
+      STATSD_KEY_PREFIX = 'worker.cst.delete_evidence_submission_records'
+
+      def perform
+        deleted_records = EvidenceSubmission.where(delete_date: ..DateTime.now).destroy_all
+        StatsD.increment("#{STATSD_KEY_PREFIX}.count", deleted_records.size)
```

Comment:
Can you add a log to record how many records were deleted and how many total evidence submissions there are here? So something like `#{name} deleted 20 / 40 EvidenceSubmission records` (might be #{class.name} instead of name`